tests/components/rflink/test_sensor.py

```diff
@@ -62,7 +67,7 @@ async def test_default_setup(hass, monkeypatch):
     new_sensor = hass.states.get("sensor.test2")
     assert new_sensor
     assert new_sensor.state == "0"
-    assert new_sensor.attributes["unit_of_measurement"] == TEMP_CELSIUS
+    assert new_sensor.attributes[ATTR_UNIT_OF_MEASUREMENT] == TEMP_CELSIUS
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)